### ISSUE TITLE
Making Hieroglyph accessible to screen reading software for the visually impaired 

### ISSUE THREAD

**Issue by dr-leo** [nyergler/hieroglyph#22]
While most web pages can be spoken with screen reading software such as Jaws for Windows, VoiceOver for Mac, I fail to read Hieroglyph slides with Jaws for windows although it generally does a good job in reading out even complex HTML such as facebook. This issue might be better addressed to the html5slides people, but I thought it doesn't hurt to mention here as well. Unfortunately, I have no idea why Jaws capitulates. 

For now my only choice is to use two computers: one for my notes and the other for the projection drawing on a sighted person to flip through the slides.

this ties in to the wish-list for the Docutils S5 writer: working with two screens simultaneously
